DOC-3132: Added  `link_attributes_postprocess` option that allows overriding attributes of a link t… (#3624)
tinymce/tinymce-docs: modules/ROOT/partials/configuration/link_attributes_postprocess.adoc

[[link_attributes_postprocess]]
== `+link_attributes_postprocess+`

> This option allows you to override attriutes of the inserted link.

Reviewer: Remove Pronoun
```suggestion
This option allows overriding attributes of an inserted link.
```
